### PR TITLE
Improve the wording of the defintion of a Counter

### PR DESCRIPTION
**Pull request by SuperQ**
* Link to the wikipedia article on Montonic functions.
* Mention that counters can be reset to zero.

### CODE REVIEW

File: content/docs/concepts/metric_types.md [hunk 1]
```diff
@@ -13,11 +13,12 @@ untyped time series. This may change in the future.
 
 ## Counter
 
-A _counter_ is a cumulative metric that represents a single numerical value
-that only ever goes up. A counter is typically used to count requests served,
-tasks completed, errors occurred, etc. Counters should not be used to expose
-current counts of items whose number can also go down, e.g. the number of
-currently running goroutines. Use gauges for this use case.
+A _counter_ is a cumulative metric that represents a single [monotonically
```
[the remark this answers is not in the document]
Comment (SuperQ) [reply content/docs/concepts/metric_types.md]: I did a bigger re-write of the wording, PTAL.

File: content/docs/concepts/metric_types.md [hunk 2]
```diff
@@ -13,11 +13,14 @@ untyped time series. This may change in the future.
...
-tasks completed, errors occurred, etc. Counters should not be used to expose
-current counts of items whose number can also go down, e.g. the number of
-currently running goroutines. Use gauges for this use case.
+A _counter_ is a cumulative metric that represents a single [monotonically 
+increasing counter](https://en.wikipedia.org/wiki/Monotonic_function) whose
+value can only increase or be reset to zero on restart. For example, you can
+use a counter to represent the number of requests served, tasks completed, or
+errors.
+
+Do not use a counter to expose a value that decreases. For example, do not use
```
Comment (brian-brazil) [inline content/docs/concepts/metric_types.md]: that can decrease

File: content/docs/concepts/metric_types.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (brian-brazil) [reply content/docs/concepts/metric_types.md]: The previous wording wasn't perfect either ;)

**Comment by brian-brazil** [comment]
:+1: 

Missing DCO

**Comment by SuperQ** [comment]
Fixed DCO.

File: content/docs/concepts/metric_types.md [hunk 3]
```diff
@@ -13,11 +13,15 @@ untyped time series. This may change in the future.
...
-tasks completed, errors occurred, etc. Counters should not be used to expose
-current counts of items whose number can also go down, e.g. the number of
-currently running goroutines. Use gauges for this use case.
+A _counter_ is a cumulative metric that represents a single [monotonically 
+increasing counter](https://en.wikipedia.org/wiki/Monotonic_function) whose
+value can only increase or be reset to zero on restart. For example, you can
+use a counter to represent the number of requests served, tasks completed, or
+errors.
+
+Do not use a counter to expose a value that can decrease. For example, do not
```
Comment (jamtur01) [inline content/docs/concepts/metric_types.md]: This is a bit picky but:

```
For example, do not use a counter for the number of currently running processes; instead use a gauge.
```

but implies preference to me whilst instead implies substitution. YMMV.

**Comment by jamtur01** [comment]
Ah sorry - missed that this was merged.

**Pull request merged**